content/en/docs/contribute/style/style-guide.md

```diff
@@ -110,8 +110,12 @@ document, use the backtick (`).
   <tr><th>Do</th><th>Don't</th></tr>
   <tr><td>The <code>kubectl run</code> command creates a Deployment.</td><td>The "kubectl run" command creates a Deployment.</td></tr>
   <tr><td>For declarative management, use <code>kubectl apply</code>.</td><td>For declarative management, use "kubectl apply".</td></tr>
+  <tr><td>For code fencing use <code>```</code>.</td><td>For code fencing use <code>" "</code>.</td></tr>
```

Comment:
I'd prefer this instead:
```
<tr><td>Enclose code samples with triple backticks. <code>(```)</code></td></tr><td>Enclose code samples with any other syntax.</td></tr>
```

Reply:
i suspect you want this

`<tr><td>Enclose code samples with triple backticks. <code>(```)</code></td><td>Enclose code samples with any other syntax.</td></tr>`

without the extra `</tr>` in `</td></tr><td>`